tests/test_executor.py

```diff
@@ -834,13 +834,15 @@ def test_exclusive_check(self, server_version, expected_message, capsys):
         ("server_version", "expected_value"),
         [(160000, None), (170000, "on")],
     )
-    def test__pg_get_metadata(self, server_version, expected_value):
+    def test__pg_get_metadata_summarize_wal(self, server_version, expected_value):
```

Comment:
```suggestion
    def test__pg_get_metadata(self, server_version, expected_value):
```

I'd say to revert this change.
I mean, we are testing more things in the test method.

Reply:
We can remove the note from the second commit when reverting this change:

```
Additionally, one of the broken tests
was renamed from `test__pg_get_metadata` to
`test__pg_get_metadata_summarize_wal` as it was noticed that it was
specifically focused on testing a summarize wal config and not the whole
of the method.
```